index.html

```diff
@@ -1,75 +1,104 @@
+  <body>
     <div class="container">
-   
-        <section class="top-content">
-            <div class="text-container">
-                <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-            <div class="text-container">
-                <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-        </section>
-        
-        <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
-            
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
+      <header>
+        <img src="img\lambda-black.png" alt="Lambda school logo" />
+        <a href="">Home</a>
+        <a href="about.html">About</a>
+        <a href="">Blog</a>
+        <a href="">Products</a>
+        <a href="">Contact</a>
+      </header>
+
+      <section class="hero-image">
+        <img src="img\jumbo.jpg" alt="highway at night" />
+      </section>
+
+      <section class="top-content">
+        <div class="text-container">
+          <h2>The Future</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+        <div class="text-container">
+          <h2>The Past</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+      </section>
 
-        <section class="bottom-content">
+      <section class="middle-content">
+        <h2>Why Did It Have To Be Boxes...</h2>
+        <div class="boxes">
+          <div class="box box-1">Box 1</div>
```

Comment:
Nice work adding 2 classes here! Another method would be to add the same class to all of the boxes and a different id to each box.